flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Agda is a dependently typed programming language / interactive theorem prover.";
+
+  inputs.flake-utils.url = "github:numtide/flake-utils";
+
+  outputs = { self, nixpkgs, flake-utils }: (flake-utils.lib.eachDefaultSystem (system: {
+    packages =  let
+      pkgs = import nixpkgs { inherit system; overlays = [ self.overlay ]; };
+    in {
+      inherit (pkgs.haskellPackages) Agda;
+
+      # TODO agda2-mode
```

Comment:
Should the TODOs be addressed before merging?

Reply:
They're opportunities for improvements, not errors. As the file stands today it's still useful and correct.